stable/memcached/README.md

```diff
@@ -40,17 +46,41 @@ The command removes all the Kubernetes components associated with the chart and
+| `image`                            | The image to pull and run                 | A recent official memcached tag                         |
+| `imagePullPolicy`                  | Image pull policy                         | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `replicaCount`                     | Number of memcached instances             | 3                                                       |
+| `pdbMinAvailable`                  | Minimum ready replicas required           | 3                                                       |
+| `AntiAffinity`                     | Hard or soft anti-affinity for Pods       | `hard`                                                  |
+| `labels`                           | Extra labels for StatefulSet/Pod/Svc      | Unset                                                   |
+| `annotations`                      | Extra annotations for StatefulSet/Pod/Svc | Unset                                                   |
+| `resources`                        | Resources requests & limits for memcached | Unset                                                   |
+| `memcached.port`                   | TCP port                                  | 11211                                                   |
+| `memcached.udpPort`                | UDP port                                  | 11211 (0 to disable)                                    |
+| `memcached.maxItemMemory`          | Max memory for items (in MB)              | `64`                                                    |
+| `memcached.maxItemSize`            | Max item size (e.g. 5m or 2k)             | `1m`                                                    |
+| `memcached.maxConnections`         | Maximum connections                       | `1024`                                                  |
+| `memcached.slabMinSizeBytes`       | Minimum slab size                         | `48`                                                    |
+| `memcached.slabGrowthFactor`       | Slab growth factor                        | `1.25`                                                  |
+| `memcached.verbosity`              | Verbosity level (v, vv, or vvv)           | `v`                                                     |
+| `memcached.extendedOptions`        | Comma-separated options                   | `modern`                                                |
+| `metrics.enabled`                  | Enable Prometheus metrics exporter        | `false`                                                 |
+| `metrics.images.repository`        | Metrics exporter repository               | `prom/memcached-exporter`                               |
+| `metrics.images.tag`               | Metrics exporter image tag                | A recent version compatible with memcached version      |
+| `metrics.images.pullPolicy`        | Metrics exporter image pull policy        | `IfNotPresent`                                          |
```

Comment:
Why is the metrics image pull policy different from the memcached image pull policy?

Reply:
In Pods each container has its own pull policy setting. I don't a view that they shouldn't be tied to the same value in the chart, but its seemed cleaner to have uniform repository/tag/pullPolicy setting for each container. And it doesn't prevent users setting both to the same value.